cloudbuild.yaml

```diff
@@ -1,11 +1,6 @@
 steps:
-- name: 'gcr.io/cloud-builders/go'
-  args: ['install', '.']
-  env: ['PROJECT_ROOT=hello']
-- name: 'gcr.io/cloud-builders/go'
-  args: ['build', 'hello']
-  env: ['PROJECT_ROOT=hello']
-artifacts:
-  objects:
-    location: 'gs://$PROJECT_ID/'
-    paths: ['hello']
+- name: golang
```

Comment:
Go mod init should only be necessary once, some time before this build is repeatedly run.

Reply:
Added a comment about this.

Reply:
I still think this should be a one-time-only step documented _outside_ the cloudbuild.yaml file.

The reason being that most users are just going to blindly copypaste whatever's in here and ignore the comments (I know I would). 

The only users who should have to do this are authors of packages since before modules, who haven't migrated to modules yet, which should hopefully be a small and shrinking set of the population.

Reply:
Documenting this outside this cloudbuild.yaml sgtm. i think we can add these instructions in the README file.